src/react/components/common/exportProviderPicker/exportProviderPicker.tsx

```diff
@@ -0,0 +1,37 @@
+import React from "react";
+import _ from "lodash";
+import { ExportProviderFactory } from "../../../../providers/export/exportProviderFactory";
+
+export interface IExportProviderPickerProps {
+    onChange: (value: string) => void;
+    id: string;
+    value: string;
+}
+
+export default function ExportProviderPicker(props: IExportProviderPickerProps) {
+    const exportProviders = _.values(ExportProviderFactory.handlers);
+
+    const allProviders = _([])
+        .concat(exportProviders)
+        .uniqBy("name")

```

Comment:
This unique expression is likely not required in here since we're not merging the providers with any other array.  In the storage/asset provider implementation there can be duplicate registrations because a "storage" provider is a "asset" provider.